R/layer_population_scaling.R

```diff
@@ -157,8 +163,8 @@ slather.layer_population_scaling <-
       ))
     }
 
-    object$df <- object$df %>%
-      dplyr::mutate(dplyr::across(tidyselect::where(is.character), tolower))
+    # object$df <- object$df %>%
+    #  dplyr::mutate(dplyr::across(tidyselect::where(is.character), tolower))
```

Comment:
I think this is wrong, but I'm not yet certain that it's safe to remove.

Reply:
Yeah, I'm not sure why it was there in the first place. I suspect that the included state population dataset used capitals while typical usage from the API gives geos in lower case. But that shouldn't have resulted in hardcoded workarounds here (that are prone to failure). 

So I think this should go forever, but I wanted to be sure that if some downstream use errored out, I could find this and try to track it more carefully.